source/tutorials.rst

```diff
@@ -70,18 +86,71 @@ BEMIO converts hydrodynamic data from WAMIT, NEMOH or AQWA into a HDF5 file, ``*
 The RM3 tutorial includes data from a WAMIT run, ``rm3.out``, of the RM3 geometry in the ``$WECSIM/tutorials/rm3/hydroData/`` directory.
 The RM3 WAMIT ``rm3.out`` file and the BEMIO ``bemio.m`` script are then used to generate the ``rm3.h5`` file. 
 
+.. note::
+    Below, a space is required between typing and ``bemio``
+
 This is done by navigating to the ``$WECSIM/tutorials/rm3/hydroData/`` directory, and typing``bemio`` in the MATLAB Command Window::
 
+
 	>> bemio
 	
 
+.. note::
+    I think this gives the false impression that this is all that would be
+    required for any BEM model when actually, if you open ``bemio.m`` there are
+    a number of steps required. Either explain what is going on here or link
+    to more detailed information elsewhere, but make it clear this is running
+    a helper file to make the tutorial quicker and that bemio.m won't work
+    on all occasions. 
+
+.. warning::
+    The first line of ``bemio.m`` is::
+        
+        clc; clear all; close all;
+    
+    Personally, I find this pretty annoying if there is something useful in
+    my workspace and a random script clears it all for me without asking. 
+    Consider removing it.
+
+.. note::
+    The ``bemio.m`` script generates a bunch of plots, but these are not
+    described at all in this tutorial, so what is the purpose of producing them?
+    Should I be looking for something?
+
 Step 2: Build Simulink Model
 """"""""""""""""""""""""""""""""""""""""""""""""
 
+.. note::
+    For someone of my basic knowledge, it would be really helpful to now ho
+    I open Simulink. Rather than have an empty file saved in the repo, would it 
+    not be more useful to take people through creating a new simulink model 
+    from scratch and then saving it as ``rm3.slx``?
+
 The WEC-Sim Simulink model is created by dragging and dropping blocks from the *WEC-Sim Library* into the ``rm3.slx`` file. 
 
+.. note::
+    How do I access the *WEC-Sim Library*?
+
 * Place two **Rigid Body** blocks from the *WEC-Sim Library* in the Simulink model file, one for each RM3 rigid body.
 
+.. note::
+    Knowing that the **Rigid Body** blocks are part of the **Body Elements** 
+    group would be helpful.
+
+.. warning::
+    When I do this I get a warning that says:
+        
+        Variable 'body' does not exist. 
+    
+    Is that normal?
+
+.. note::
+    In the diagram these blocks have the names "Float" and "Spar/Plate". How
+    did that happen?
+
```

Comment:
You can rename a block in Simulink, but it isn't a necessary step

Reply:
I think it can be a bit confusing to have images that are not what you get when you follow the instructions.